packages/ui/core-components/src/lib/unsorted/viz/table/_DataTable.svelte

```diff
@@ -44,9 +44,18 @@
 	$: rowNumbers = rowNumbers === 'true' || rowNumbers === true;
 
 	// Sort props
-	export let sortBy = undefined;
-	export let sortAsc = false;
-	$: sortAsc = sortAsc === 'true' || sortAsc === true;
+	export let sort = undefined;
+	let sortBy = undefined;
+	let sortAsc = undefined;
+	$: if (sort) {
+		const [column, direction] = sort.split(' ');
+		sortBy = column;
+		sortAsc = direction === 'asc' ? true : false; // Default to descending if no direction is provided
```

Comment:
this is the opposite to SQL standard

Reply:
Hmm yea I don't like that asc is default, but I guess if we move to sql snippets this will get confusing. Will update this